extension/js/inspector/app/modules/Activity/views/ActionList.js

```diff
@@ -0,0 +1,51 @@
+define([
+  'backbone',
+  'marionette',
+  'util/Radio', // ?
+  "text!templates/devTools/activity/action.html",
+  'app/behaviors/ClickableProperties', // ?
+  'util/isViewType', // ?
+], function(Backbone, Marionette, Radio, tpl, ClickableProperties, isViewType) {
+
+  var ChildView = Backbone.Marionette.ItemView.extend({
+    template: tpl,
+    ui: {
+      toggleBtn: '[data-action="toggle"]'
+    },
+    events: {
+      'click @ui.toggleBtn': 'onClickToggle'
+    },
+    onClickToggle: function() {
+      this.model.isCollapsed = !this.model.isCollapsed;
+      this.toggleNode();
+      return false;
```

Comment:
So, do we want to show the tree below the action like this, or replace the action list with the tree. I'm good with either behavior